src/_common/community/activity-item/activity-item.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+	<div class="-item" :class="{ '-item-usersplit': usersplit }">
+		<span v-if="!Screen.isXs" class="-left">
+			<app-user-avatar class="-avatar" v-if="usersplit" :user="item.user" />
+			<span v-else class="-time">
+				{{ item.added_on | date('shortTime') }}
+			</span>
+		</span>
+
+		<div class="-main">
+			<div v-if="usersplit" class="-usersplit">
+				{{ item.user.display_name }}
+				<span class="-user-sub">
+					<template v-if="!Screen.isXs"> @{{ item.user.username }} </template>
+					<span
+						class="-user-sub-date"
+						v-app-tooltip="
+							date(item.added_on, 'fullDate') + ' ' + date(item.added_on, 'shortTime')
+						"
+					>
+						<span v-if="isToday">
+							<translate>Today</translate>
+						</span>
+						<span v-else-if="isYesterday">
+							<translate>Yesterday</translate>
+						</span>
+						<span
+							v-if="isToday || isYesterday"
+							v-translate="{ time: date(item.added_on, 'shortTime') }"
+						>
+							at %{ time }
+						</span>
+						<span v-else>
+							{{ date(item.added_on, 'shortDate') }}
+						</span>
+					</span>
+				</span>
+			</div>
+			<div class="-content">
+				<span
+					class="-icon"
+					:class="{
+						'-icon-notice': color === 'notice',
+						'-icon-theme': color === 'theme',
+						'-icon-other': color !== 'notice' && color !== 'theme',
+					}"
+				>
+					<app-jolticon v-if="shouldShowIcon" :icon="icon" />
+				</span>
+
+				<div class="-action">
+					<span v-if="item.type === 'community/created'" v-translate>
+						<em>Created</em> this awesome community.
+					</span>
```

Comment:
I thought we were phasing away v-translates. do we have a plan for sanitizing these?

Reply:
Oh I wasn't aware, why are we not doing v-translate?
Should we just do <translate> tags instead?

And what do you mean with "sanitize"?